backend/models/submission.js

```diff
@@ -0,0 +1,81 @@
+const mongoose = require("mongoose");
+
+const matrix = new mongoose.Schema({
+  x: {
+    type: String
+  },
+  y: {
+    type: String,
+  }
```

Comment:
These should probably be type: Number not string

Reply:
Also don't need a comma after the type: String on line 8

Reply:
Probably why the CI test is failing

Reply:
Makes sense! Just made the quick fixes. My formatter wasn't working properly for a bit which is how some of the errors snuck in.